### DISCUSSION TITLE
Chia c sürücüsünü dolurdu

### DISCUSSION THREAD

**Discussion by FatihAyturk** [Chia-Network/chia-blockchain#17636] [Support]
Merhaba, chia  c sürücüsünü doldurdu. nasıl temizlenir.

**Comment by BrandtH22** [comment]
Hey @FatihAyturk , as a heads up we can generally provide more timely and thorough support on our discord server (https://discord.gg/chia). 

Firstly we recommend the blockchain db is stored on a fast SSD (520Mb/s or faster) but you can move the drive that teh db is stored on by following these directions:
https://docs.chia.net/faq/#how-do-i-move-the-chia-blockchain-database-to-another-drive

Do let us know if any questions arise in the process :+1:

**Reply by BrandtH22** [reply]
Not: Bu çeviri yazılımı kullanılarak çevrilmiştir

Hey @FatihAyturk, bir uyarı olarak genellikle discord sunucumuzda (https://discord.gg/chia) daha zamanında ve kapsamlı destek sağlayabiliriz. İlk olarak, blok zinciri veritabanının hızlı bir SSD'de (520 Mb/sn veya daha hızlı) depolanmasını öneririz, ancak aşağıdaki talimatları izleyerek veritabanının depolandığı sürücüyü taşıyabilirsiniz: https://docs.chia.net/faq/#how-do-i-move-the-chia-blockchain-database-to-another-drive Bu süreçte 👍 herhangi bir sorunuz olursa bize bildirin

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
